cmd/internal/flowmode/cmd_run.go

```diff
@@ -99,19 +101,23 @@ depending on the nature of the reload error.
 		StringVar(&r.clusterJoinAddr, "cluster.join-addresses", r.clusterJoinAddr, "Comma-separated list of addresses to join the cluster at")
 	cmd.Flags().
 		BoolVar(&r.disableReporting, "disable-reporting", r.disableReporting, "Disable reporting of enabled components to Grafana.")
+	cmd.Flags().StringVar(&r.configFormat, "config.format", r.configFormat, "The format of the source file. Supported formats: 'prometheus'.")
```

Comment:
Isn't default config format `river`? Should we make it the default and a supported option? I think this makes sense for a flag called `config.format`.

Alternatively, if we want this flag to mean "convert config from this format to river", then we could name the flag differently, e.g. `config.convert-format`".

Reply:
+1 to `config.convert-format`. @rfratto do we have consensus on this?

Reply:
`-config.format` defaulting to `river` (ie apply no conversion) is a good idea. 

I'm not sure about `config.convert-format`, to me it implies converting TO a format, rather than converting FROM a format. 

I don't want to block on bikeshedding though, final call is yours.

Reply:
I can update the default and code it to handle `river` as existing behavior.

Reply:
> I can update the default and code it to handle river as existing behavior.

I don't think that's necessary, but the CLI could understand river as "no op", and not-river as "do a conversion"


BTW: I don't think it should be `river`, but `flow` instead; River is the language, Flow is the expected schema of a config file written in that language. It's the same reason one of the expected formats is Prometheus (the schema) and not YAML (the language)

Reply:
makes sense. I have to add 1 small conditional to not call the converter if `flow` otherwise it will pass it along.

Reply:
This change has been implemented. I'll merge this tomorrow pending any last minute feedback.

Reply:
LGTM 👍